### ISSUE TITLE
Automatically convert tag formats between sources

### ISSUE THREAD

**Issue by Bionus** [Bionus/imgbrd-grabber#1100]
Some sources use different tag format's, for example `some_tag another_tag`, `Some Tag + Another Tag`, `Some_tag && Another_tag`, etc.

The program be able to make a single multi-site search by automatically converting the case, work separators and tag separators. Also, the user should be able to tell which is his "favorite" format that the program should use to convert searches from.

:warning: It must be possible to disable it on demand or globally for some more complex queries.

**Comment by MasterPetrik** [comment]
Also may be connected with https://github.com/Bionus/imgbrd-grabber/issues/916

**Comment by Bionus** [comment]
Yes, related, but that one is for display of tags while here is for search only (so it also includes separators).
Standardization applies to both though. :+1:
